config/src/config/network_config.rs

```diff
@@ -260,7 +248,17 @@ impl Identity {
     pub fn peer_id_from_config(&self) -> Option<PeerId> {
```

Comment:
oh, I actually meant we should delete this snippet.

My recommendation is if you can remove this latest commit, we can land this pr... otherwise eliminate this function.

Reply:
Okay, I just removed that function. Right now there's an `.expect("identity key should be present")` call in the `peer_id()` method, I'm not sure if there'd be any use cases where a return type of `Option<PeerId>` would be more desirable.

Reply:
Definitely something to contemplate! I think we'll need to do some improvements in the network code base to catch up.